app/templates/components/job-input-file-list.hbs

```diff
@@ -8,8 +8,8 @@
   </tr>
   {{#each files as |file|}}
     <tr>
-      <td>{{file.destinationPath}}</td>
-      <td>{{file.size}}</td>
+      <td class="file-destination-path">{{file.destinationPath}}</td>
+      <td class="file-size">{{number-to-human-size file.size true 2}}</td>
```

Comment:
Why did you switch from traditional to IEC Byte Units? (the `true` bit)

Reply:
I would have used this from the start, but didn't notice the helper supported the option. The IEC byte units are consistent with both the [Duke DS web portal](https://github.com/Duke-Translational-Bioinformatics/duke-data-service-portal/blob/67fe6847dacb344cb472c8591ac2c813502c1ad8/app/scripts/util/baseUtils.js#L6-L9) and [ls -lh](http://man7.org/linux/man-pages/man1/ls.1.html)

Reply:
I think you have that backwards (or I have managed to reverse them in my head). 

__Duke DS web portal:__
<img width="434" alt="screen shot 2017-10-02 at 1 54 55 pm" src="https://user-images.githubusercontent.com/1024463/31091549-59dd367e-a779-11e7-8a72-781965ff68a2.png">


__ls -lh__
```
-h, --human-readable
              with -l and/or -s, print human readable sizes (e.g., 1K 234M
              2G)
```

Reply:
Though DukeDS displays **GB** instead of **GiB**, it still divides byte counts by powers of 1024 to get human-readable sizes (see above link). **ls -lh** does too (see below)

Enabling IEC Byte units in [number-to-human-size](https://www.npmjs.com/package/ember-number-to-human-size#usage) makes the divided numbers consistent with the web portal and ls -lh.

Test confirming expected behavior:
https://github.com/Duke-GCB/bespin-ui/blob/cleanup-job-summary/tests/integration/components/job-input-file-list-test.js#L35

Also from **ls -lh**
```
       -h, --human-readable
              with -l and/or -s, print human readable sizes (e.g., 1K 234M
              2G)

       --si   likewise, but use powers of 1000 not 1024
```

Reply:
Sounds good. I was just looking at the letters `GB` vs `GiB`.